docs/user-guide/station-authentication-modes.md

```diff
@@ -0,0 +1,132 @@
+
+Same can be done for a cups.trust file by overriding the **'CUPS_TRUST_PATH'** environment variable
+
+## Client authentication
+
+The LoRaWan Network Server implementation provided by this starter kit is allowing client authentication from a Basics Station client.
+
+### Changing 'Client Certificate Mode' in 'LoRaWan Network Server module'
+
+By default, the 'LoRaWanNetworkSrvModule' is not accepting/requiring any client certificate for its exposed endpoints.
+
+It is possible to modify the behavior of the module to actually either allow or require a client certificate for accessing to its endpoints.
+
+For example, to always require a client certificate for reaching LNS/CUPS endpoints, you should set the **'CLIENT_CERTIFICATE_MODE'** environment variable in 'LoRaWanNetworkSrvModule' to a value of '2' or 'RequireCertificate'.
+
+Supported values for 'CLIENT_CERTIFICATE_MODE' environment variables are:
+
+- `0` or `NoCertificate`: client certificate is not required and will not be requested from clients
+- `1` or `AllowCertificate`: client certificate will be requested; however, authentication will not fail if a certificate is not provided by the client
+- `2` or `RequireCertificate`: client certificate will be requested, and the client must provide a valid certificate for authentication to succeed
+
+### Importing 'tc.crt/tc.key/cups.crt/cups.key' in bundled 'LoRaBasicsStationModule'
+
+If you are making use of the bundled 'LoRaBasicsStationModule', it's possible to import a client certificate (.crt + .key files) in the module itself.
+
+The default path where the files will be searched is '`var/lorastarterkit/certs/`. As described in the previous section, it is possible to bind a folder on the host OS to the one mentioned above.
+
+The default path of the tc.crt/tc.key/cups.crt/cups.key file can be overridden by using the related environment variable (i.e.: setting any of **'TC_CRT_PATH'**, **'TC_KEY_PATH'**, **'CUPS_CRT_PATH'**, **'CUPS_KEY_PATH'** to `/var/otherfolder/my.crt` will make the module copy the my.crt file to a *.crt in the LoRa Basics™ Station working directory).
+
+### Providing a list of allowed client thumbprints for connection
+
+At the server side, the validation of the certificate is happening by comparing the thumbprint of the certificate provided for authentication with a list of allowed thumbprints stored in the Concentrator Twin (more information on 'clientThumbprint' property of Twin in [related ADR](https://azure.github.io/iotedge-lorawan-starterkit/dev/adr/006_cups/))
+
+When using the provided **Cli-LoRa-Device-Provisioning** tool to provision a concentrator device to IoT Hub, you can pass the **'--client-certificate-thumbprint'** option to specify the thumbprint of an allowed certificate.
```

Comment:
nit: the `--client-certificate-thumbprint` is not documented on Tools page

Reply:
I can see it

Reply:
🙈